db.json

```diff
@@ -6369,6 +6373,10 @@
     "compressible": true,
     "extensions": ["vtt"]
   },
+  "text/x-ansi": {
+    "compressible": true,
+    "extensions": ["ans"]
+  },
```

Comment:
That's probably why I was gravitated towards this PR as it sparked some curiosity and old neurons. ;) Definitely miss my BBS days.